panel/lab/components/fields/radio/index.vue

```diff
@@ -1,3 +1,90 @@
 <template>
-	<k-lab-options-field-examples type="radio" />
+	<k-lab-form>
```

Comment:
Something is strange here and I don't know what it is, but the examples without columns don't have a proper width for each option.

Reply:
Is that different to the current live lab?

Reply:
You are right, it actually isn't